keadm/cmd/keadm/app/cmd/reset.go

```diff
@@ -78,6 +81,18 @@ func NewKubeEdgeReset(out io.Writer, reset *types.ResetOptions) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if !reset.Force {
+				fmt.Println("[reset] WARNING: Changes made to this host by 'keadm init' or 'keadm join' will be reverted.")
+				fmt.Print("[reset] Are you sure you want to proceed? [y/N]: ")
+				s := bufio.NewScanner(os.Stdin)
+				s.Scan()
+				if err := s.Err(); err != nil {
+					return err
+				}
+				if strings.ToLower(s.Text()) != "y" {
+					return fmt.Errorf("aborted reset operation")
+				}
```

Comment:
Do we need to have some retry like 3 times here?

Reply:
Similar action as `kubeadm`, just add one prompting for confirmation.